### ISSUE TITLE
Screenreader reading French text in Notify app (p class) in English [M]

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#928]
**Done When**
- Screen readers Parlez Français

This is happening on the homepage for “Essayez Notification, une nouvelle plateforme de messagerie pour le gouvernement du Canada”

It also happens throughout the site. Ideally, French text is read in French voice.

Check language declaration in HTML
Language attribute = EN?
Should be FR-CA on French pages?

https://docs.google.com/document/d/18W-nKAH-54wpNy5XSZeEA31my4XRMxpZRVJ3en4ThAA/edit

https://docs.google.com/document/d/1JaJzEnQ7M6tQRziwEC1q8_UUQBizAdVRtUashXdYZeo/edit

**Comment by smcmurtry** [comment]
I tried to reproduce this with voiceover but I couldn't - “Essayez Notification, une nouvelle plateforme de messagerie pour le gouvernement du Canada” was read with a french accent for me.

The only french content on the homepage that was not read with a french accent was the alt-text for images. Maybe that is fixable by adding a `lang` attribute to the images?

**Comment by sastels** [comment]
From http://jkorpela.fi/html/alt.html:
> According to WAI guidelines, an author should indicate the language of each element on the page. This can be done using the HTML attribute lang, or the XML attribute xml:lang, or both. It suffices to set such an attribute for the document as a whole, in the <html> tag, and, when needed, for each element that is in a language different from the main language or, more exactly, different from the language of its enclosing element. Thus, if your document is in one language only, then you have an easy task.

So it should read the alt text correctly if the page `lang` is set correctly, but who knows what's going on here 🤷

**Comment by brdunfield** [comment]
Nothing is being read with a french accent on the homepage with my VoiceOver.

**Comment by brdunfield** [comment]
With Safari I was able to get my VoiceOver to correctly read the page in French. I was unable to reproduce this bug. The indicated paragraph was read correctly in French with a french accent.

Image alt-text was also read correctly in French with a french accent.
